.github/workflows/publish-to_npm.yml

```diff
@@ -9,9 +11,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       # Setup .npmrc file to publish to npm
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v3
         with:
-          node-version: "12.x"
+          node-version: "18.x"
```

Comment:
Again, I have no way to confirm this workflow will still work after this change.